### ISSUE TITLE
New failures in last night cron CI jobs

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#4279]
These three tests failed in yesterday CI jobs:
```
	180 - doc/examples/ex11/ex11.sh (Failed)
	411 - test/grdfill/splinefill.sh (Failed)
	445 - test/grdimage/rgb_grids.sh (Failed)
```

**Comment by seisman** [comment]
These failures are caused by changes in PR #4276.

**Comment by seisman** [comment]
The above failures are fixed in #4280, but I'm having other new failures.
```
	202 - doc/examples/ex33/ex33.sh (Failed)
	481 - test/grdtrack/layout.sh (Failed)
	923 - test/spotter/spotter_09.sh (Failed)
	924 - test/spotter/spotter_10.sh (Failed)
```
Can you reproduce these new failures locally?

**Comment by PaulWessel** [comment]
OK, instead of getting 160+ failures and hunt for the real failures, I am doing this by plotting module first (ctest -R grdimage) until actual failures are removed and then it is mostly failures due to change of policy.

**Comment by joa-quim** [comment]
Yes, I get those failures too.

**Comment by joa-quim** [comment]
But if I run those scripts directly, the result is OK. Don't know why the tests failed without even creating the PNGs.

**Comment by PaulWessel** [comment]
I also do not understand why a few pscoast plots show changes along coastlines. Maybe pscoast sets a certain join and I now inserted a gridline call that changed that....

**Comment by PaulWessel** [comment]
These failures were fixed via #4283, closing this issue.
